lib/bulk.js

```diff
@@ -83,6 +83,17 @@ class Bulk {
     const setResult = (cmd, cmdResult) => {
       const cmdKey = Object.keys(cmd)[0];
       result[this.cmdKeys[cmdKey]] += cmdResult.n;
+      // Collate and report writeErrors and writeConcernErrors to the result object.
+      if (cmdResult.writeErrors) {
+        for (const writeError of cmdResult.writeErrors) {
+          result.writeErrors.push(writeError);
+        }
+      }
+      if (cmdResult.writeConcernErrors) {
```

Comment:
Let's drop this block for now, since the driver throws on write concern errors.

Reply:
https://github.com/mongoist/mongoist/pull/48/commits/aee3b185c4e16258fa201a7b85fb3738d50f61b9